css/css-ui/appearance-button-other-form-controls-001-ref.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Reference for CSS Basic User Interface Test: appearance: button on form controls other than select</title>
+<style>
```

Comment:
As per https://github.com/web-platform-tests/wpt/pull/14993/files#r250017177, please explicitly add `appearance: auto` to this ref file, since it isn't the initial value of the property. Otherwise, we're testing the content of the UA stylesheet, which is a valid, but separate, thing to do.